package.json

```diff
@@ -29,6 +29,15 @@
     "seed:configdb": "node src/connectors/configDb/configDb-seed/runSeedScripts",
```

Comment:
src/connectors/configDb/configDb-seed/runSeedScripts.js needs updating too?

Reply:
I'm not even sure what these are to be honest, maybe the automated way of adding new councils to the service, don't think we've used them for a while.

Reply:
Updated this with the new collection name  anyway, but on the release branch